CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ All notable changes to this project will be documented in this file.
 Wazuh commit: TBD \
 Release report: TBD
 
+## Fixed
```

Comment:
This entry is duplicated

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/4523/commits/d599dd8db741eae95fd4d6d758c155a45edd98cb